src/mcp/tools/dataconnect/get_dataconnect_schema.ts

```diff
@@ -0,0 +1,38 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { toContent } from "../../util.js";
+import * as client from "../../../dataconnect/client";
+
+export const get_dataconnect_schema = tool(
+  {
+    name: "get_dataconnect_schema",
+    description:
+      "List the Firebase Data Connect Schema in the project, which includes Cloud SQL data sources and the GraphQL Schema describing what tables are available.",
+    inputSchema: z.object({
```

Comment:
Should there be an option like `include_generated: z.boolean()` that when true fetches the full SDL including all the stuff we add automatically?

I don't think it should be the default, but maybe should be something that's described as "use this to get the full GraphQL schema including automatically generated items if you are having issues constructing valid queries"